build.gradle

```diff
@@ -19,8 +19,16 @@ buildscript {
 }
 
 ext {
-    mockito_version = '3.11.0'
+    junit_version = '4.13.2'
+    mockito_version = '3.11.2'
+    androidx_test_rules_version = '1.4.0'
+    androidx_test_runner_version = '1.4.0'
+    androidx_test_ext_junit_version = '1.1.3'
+    androidx_test_espresso_espresso_core_version = '3.3.4'
```

Comment:
@Endorf As we use this ext for espresso core and intents I think we should rename this to "androidx_test_espresso_version"